.github/generate-envs.py

```diff
@@ -220,11 +220,20 @@
     {
         "lang": "verilog",
         "sim": "verilator",
-        "sim-version": "stable",
+        "sim-version": "master",
         "os": "macos-13",
         "python-version": "3.8",
         "group": "experimental",
     },
+    # Verilator macOS
```

Comment:
nit (to distinguish from the one above):

```suggestion
    # Verilator macOS latest release
```